pkg/sync/manager/controller.go

```diff
@@ -54,18 +55,28 @@ type SyncController struct {
 	WatchedGVKs    []schema.GroupVersionKind
 	Queue          *sqs.Config
 	ResourceParser *parser.ResourceParser
+	Metrics        monitoring.MetricsI
 }
 
 func (c *SyncController) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	var start = time.Now()
 
 	log := c.Log.WithValues("name", req.Name, "namespace", req.Namespace)
 	log.Info("start")
-	defer log.Info("end", "duration", time.Since(start))
+	defer func() {
+		elapsed := time.Since(start)
+		log.Info("end", "duration", elapsed)
+		// TODO Should I try to get the cluster name out of this? Or is the sync object name good enough?
```

Comment:
The object name should be good enough (since there ideally should be one object for each cluster)